.pre-commit-hooks.yaml

```
@@ -13,7 +13,7 @@
     Rewrites all Terraform configuration files to a canonical format.
   entry: hooks/terraform_fmt.sh
   language: script
-  files: (\.tf|\.tfvars)$
+  files: (\.tf|\.tfbackend|\.tfvars)$
```

Just a nit to improve regex overall while we're here:
```suggestion
  files: \.tf(backend|vars)?$
```